src/documents/IoCAPI.md

```diff
@@ -167,3 +183,221 @@ component member name.
+evaluating all component options, members, invokers and other top-level masterial, and the latter marks the component
+as fully constructed and fires its `onCreate` event.
+
+The definitions from `fluid.modelComponent` look as follows:
+
+```javascript
+fluid.defaults("fluid.modelComponent", {
+    gradeNames: ["fluid.component"],
+
+    workflows: {
+        global: {
+            resolveModelSkeleton: {
+                funcName: "fluid.resolveModelSkeleton"
+            }
+        },
+        local: {
+            notifyInitModel: {
+                funcName: "fluid.notifyInitModel",
+                priority: "before:concludeComponentInit"
+            }
+        }
+    }
+// ...
+});
+```
+
+This shows the registration of one global workflow, `fluid.resolveModelSkeleton` which since it is the only one in the core
+framework has no priority annotation. It also shows the registration of a local workflow, `fluid.notifyInitModel` which
+is responsible for firing any [model listeners](ChangeApplierAPI.md#declarative-style-for-listening-to-changes) which need
+to be notified of initial model values as part of the [initial transaction](ModelRelay.md#the-initial-transaction). The
+[priority](Priorities.md] field attached here indicates that this should happen (if no further definition intervenes) immediately
+before the `concludeComponentInit` action which fires `onCreate`.
+
+### Global workflows
+
+There are two kinds of workflow functions which may be contributed, global and local workflows. Global workflows represent
+particularly ambitious functionality which requires oversight of the entire instantiating tree, and all global workflows
+execute before any local workflows. The framework currently includes just one global workflow, the one which identifies
+and resolves the values of all model components throughout the instantiating tree's `model skeleton`, but further ones
+will be implemented as part of the upcoming framework's markup renderer for Infusion 5.0 as described in
+[FLUID-4260](https://issues.fluidproject.org/browse/FLUID-4260).
+
+A global workflow function receives the signature:
+
+`<global.workflow.function.name>(shadows)` where
+
+* `shadows {Array of Shadow}` An array of _all_ component shadows participating in this transaction, sorted in order
+    from the root of the component
+    tree down to the leaves. Note that this includes all such components, and the implementor will need to make an
+    explicit [fluid.componentHasGrade](CoreAPI.html#fluidcomponenthasgradecomponent-gradename) check in order to
+    find the components of interest to it. A shadow
+    record includes various book-keeping fields of interest to the framework, but of primary interest is the member
+  * `that {Component}` The currently instantiating component. Much of the component's material at this point will be
+    unevaluated. Any material which is required by the component must have its observation triggered via
+    `fluid.getForComponent`.
```

Comment:
Is this intended to be a list with two levels and one item at each level? It's a bit hard to read in the current format.

Reply:
Yes, it is intended to be indented since `that` is a member of each `shadow` array element